central/debug/service/service.go

```diff
@@ -598,6 +620,30 @@ func (s *serviceImpl) writeZippedDebugDump(ctx context.Context, w http.ResponseW
 
 	fetchAndAddJSONToZip(debugDumpCtx, zipWriter, "system-configuration.json", s.getConfig)
 
+	// Fetch alerts and append to the zip
+	if opts.withAlerts {
+		fetchAndAddJSONToZip(debugDumpCtx, zipWriter, "alerts-before.json", s.fetchAlerts)
+		// clusterSet := set.NewFrozenStringSet(opts.clusters...)
+		for _, conn := range s.sensorConnMgr.GetActiveConnections() {
+			err := conn.InjectMessage(debugDumpCtx, &central.MsgToSensor{
+				Msg: &central.MsgToSensor_ReprocessDeployments{
```

Comment:
I wonder if we are OK with introducing side effects to the diag bundle, and whether we shouldn't rather introduce completely separate flow for that

Reply:
I agree. But adding a new flow might be inconvenient for customers. So, our proposal here is to add a flag that is disabled by default in diag bundles and will only be used for this specific case. 
```
roxctl central debug download-diagnostics --alerts=true --timeout=100
```

Ideally, we should remove this after this release, once re-sync is disabled by default on customer's clusters.  

Would you suggest adding a new `roxctl debug` command altogether for this?